src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java

```diff
@@ -357,14 +404,6 @@ void singleLineReturnNoSpace() {
               java(
                 """
                   class Test {
-                      /**@return*/
-                      int method() {
-                      }
-                  }
-                  """,
-                """
-                  class Test {
-                      /***/
```

Comment:
With these lines removed the test no longer verifies anything; is this a case that was hard to cover? Should the test be removed instead?

Reply:
TBH, I totally forgot, why I removed these. Maybe by accident?